### DISCUSSION TITLE
Can we make exports keep the low quality preview? Macs are funny about h265 codec

### DISCUSSION THREAD

**Discussion by dopeytree** [blakeblackshear/frigate#13111] [General]
My cameras are set to h265 to save on storage. 

Apple m3 with Safari 17.6 seems to only play h265 with a h1c tag.

Is there a way we can keep the low quality previews that frigate uses as fall back etc?

<img width="1512" alt="Screenshot 2024-08-15 at 22 16 14" src="https://github.com/user-attachments/assets/69eeb285-98ba-4bda-9615-338d3ca70f18">
<img width="1512" alt="Screenshot 2024-08-15 at 22 15 49" src="https://github.com/user-attachments/assets/706b82ff-d23f-4420-bd4c-de5e64557e16">

**Comment by NickM-27** [comment]
We can probably change the tag while creating the export, I don't believe that causes issues but would have to see
